src/tailwind.css

```diff
@@ -0,0 +1,19 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+h1 {
```

Comment:
Ignore that these selectors are a bad idea—I just added some basic styles to page elements to get up and running and will do something more encapsulated in future.